### DISCUSSION TITLE
What does the sensor.load_power entity include / exclude?

### DISCUSSION THREAD

**Discussion by calum-mcfarlane** [nathanmarlor/foxess_modbus#419] [Q&A] [answered]
A bit of pre-amble to explain why I'm asking the question: I've noticed that the "idle" power draw in our house rarely drops below 200W, even when all lights are turned off, etc etc. There _are_ things that run all the time (Wifi APs, proxmox host, ethernet switches) as well as things that run periodically (fridge, freezer, washing machine etc), but I have all of those connected via LocalBytes plugs so I can see their usage.

The other day I created a HA 'helper' group that adds up all the things that **are** being measured, and a template sensor that subtracts the resulting "Total Measured Power" from the load_power value. This gives me a "Non-Measured Power" value which seems to be about 100W, +/- 20W, even when I've switched off every light, wall socket etc I can think of. Given that this adds up to ~2.4kWh every day, I'm keen to track it down and eliminate it where possible - 2.4kWh is about equivalent to a whole HV2600 module worth of energy!

So, my question, before I start chasing down this 'mystery' 100W, is trying to sense check the basic info - ie, **is the load_power sensor all real, genuine power draw in my house?** If I theoretically disconnected everything and stood in front of the inverter front panel, should I see "Load=0W" on there? I'm aware there are losses in converting from DC to AC ie from batteries to "mains" or whatever, and I have a separate "System Losses" sensor that attempts to quantify this (left over from when I was using the StealthChesnut integration). However, as far as I can think, that's all happening "upstream" of the actual demand for power from the mains wiring in the house, so it's not relevant to my missing 100W - right?

**Comment by nathanmarlor** [comment]
Maybe one for the Facebook group for wider input but my first guess would be the inverter itself - they're quite power hungry!

**Reply by jamesog** [reply]
As someone who doesn't use Facebook I'd love to see an answer posted somewhere more accessible :-)

**Reply by calum-mcfarlane** [reply]
Yeah, I've got a FB Account but I don't use it. Keeping it on here for now (I will post to FB about if I really have to!)

The YAML for my losses sensor is 

```  
  - name: "System Losses"
        unique_id: foxess_inv1_system_losses
        device_class: "power"
        unit_of_measurement: "kW"
        state: >
               {{ ((states('sensor.pv1_power') | float(default=0)
               + states('sensor.pv2_power') | float(default=0)
               + states('sensor.grid_consumption') | float(default=0)
               + states('sensor.battery_discharge') | float(default=0)
               - states('sensor.battery_charge') | float(default=0)
               - states('sensor.feed_in_power') | float(default=0)
               - states('sensor.load_power') | float(default=0) )) | round(2) }}
```

It's usually in the ~50-70W range, but can be double that when the inverter is working hard. I suppose these things, like all power supplies, are less efficient at low loads.

I still find it a little odd that the inverter's own losses would be seen as a 'load' by its own sensor, but if that's how it works, that's how it works (and the fact the losses vary with the output of the inverter very much suggests it is how it works!)

So maybe I need to add the value of my losses sensor to the "measured power" group, so that I am looking at the remaining consumption...

**Comment by FozzieUK** [comment, marked as answer]
@calum-mcfarlane yes you are correct the house load is all real house load being consumed in your house as it’s measured at the incoming meter - but, i’ll qualify that whilst it is all of the load being drawn in the house, the inverter itself requires anywhere between 70 and 120w to stay operational (as Nathan said they are quite power hungry - the 3 phase inverters are much worse - nearer 200w) and that is being included in your house load.

The demand is always met by solar or the batteries discharging (and a small trickle from the grid), that’s why you tend to see a continuous 120w discharge on the battery at night (effectively the batteries take the entire 100w of ‘inverter’ house load plus the vbus conversion losses).

I wouldn’t work too hard to find that missing 100w it’s mostly consumed by the inverter ;)

**Reply by calum-mcfarlane** [reply]
Thanks @FozzieUK, I'm slightly surprised it's that high, but then again we're talking about something that can go from tens of watts to thousands in a matter of a tenth of a second or so, which is pretty remarkable when you think about it. 

You've also answered another question I had which was about that 120W draw, which I had not realised was from the inverter itself (although it makes complete sense). 

Out of curiosity I'll see what the numbers look like when I subtract the losses from the "Non-Measured Load", there's things like the house alarm and probably a few other odds and ends that will make up the 30W or so of whatever is left over. As you say, not worth getting too hung up about.

**Reply by FozzieUK** [reply]
All inverters have a no-load consumption, it takes quite a bit of effort to create the waveforms just in case a load turns up - the hybrid’s seem to be a bit more power hungry than solar only (30-50w), possibly from having to keep the BMS and battery side running as well.
